README.md

```diff
@@ -15,13 +15,111 @@
 
 ## Usage
 
-> Some usage instructions
+> Please run the following scripts to start this service:
```

Comment:
Wow, seeing this Readme on Github is really amazing!!! I'll definitely have to mimic you in this regard!! Hopefully more people get to see and admire what you've done.